ansible/library/announce_routes.py

```diff
@@ -150,6 +151,7 @@ def change_routes(action, ptf_ip, port, routes):
     url = "http://%s:%d" % (ptf_ip, port)
     data = {"commands": ";".join(messages)}
     # nosemgrep-next-line
+    time.sleep(0.01)
```

Comment:
Instead of using fixed sleep here, maybe add some logic to retry. To avoid more retries overwhelming the HTTP server, you can use "backoff" algorithm. This means if the first try failed, wait some time to try again. If the 2nd try failed, increase wait interval and continue trying. After maximum retries reached, stop retrying and fail explicitly.